Makefile

```diff
@@ -69,6 +69,10 @@ update-crds: ## Update CRDs
 	kustomize build config/crd/app-protect-dos --load-restrictor='LoadRestrictionsNone' >deploy/crds-nap-dos.yaml
 	kustomize build config/crd/app-protect-waf --load-restrictor='LoadRestrictionsNone' >deploy/crds-nap-waf.yaml
 
+.PHONY: generate-manifests
+generate-manifests: ## Generate manifests
+	./hack/generate-manifests.sh
+
```

Comment:
All looks great on my end

I wanted to add one suggestions, but it's non-blocking
It might be useful to add another make command here for generating just a single deployment manifest.

For example
```
TYPE ?= default
.PHONY: generate-manifest
generate-manifest: ## Generate a single manifest
	helm template nginx-ingress --namespace nginx-ingress --values examples/helm-chart/$(TYPE)/values.yaml charts/nginx-ingress > deploy/$(TYPE)/deploy.yaml
```

Reply:
What's the use case?